src/routes/comments/index.ts

```diff
@@ -0,0 +1,18 @@
+import express from 'express'
+import expressWrapper from '../../helpers/expressWrapper'
+import CommentsController from '../../controllers/CommentsController'
+import checkUser from '../../middlewares/checkUser'
+
+const router = express.Router()
+router.get('/events/:eventId/comments'
+  , expressWrapper(CommentsController.index))
+router.post('/events/:eventId/comments', expressWrapper(checkUser)
+  , expressWrapper(CommentsController.store))
+router.get('/events/:eventId/comments/:id',
+  expressWrapper(CommentsController.show))
+router.put('/events/:eventId/comments/:id'
+  , expressWrapper(CommentsController.update))
+router.delete('/events/:eventId/comments/:id'
+  , expressWrapper(CommentsController.destroy))
```

Comment:
If I were you I will go with something like this to avoid repetition 
```suggestion
const router = express.Router()
router.get('/events/:eventId/comments'
  , expressWrapper(CommentsController.index))
router.get('/events/:eventId/comments/:id',
  expressWrapper(CommentsController.show))

router.use(expressWrapper(checkUser)) // protected routes below

router.post('/events/:eventId/comments',
  expressWrapper(CommentsController.store))
router.put('/events/:eventId/comments/:id'
  , expressWrapper(CommentsController.update))
router.delete('/events/:eventId/comments/:id'
  , expressWrapper(CommentsController.destroy))
```